conjure-java-jersey-server/src/main/metrics/conjure-server-metrics.yml

```diff
@@ -0,0 +1,12 @@
+options:
+  javaPackage: com.palantir.conjure.java.server.jersey
+  javaVisibility: packagePrivate
+namespaces:
+  server.response:
+    shortName: ConjureServer
+    docs: Conjure server metrics
```

Comment:
Metrics produced by Jersey servers using the `ConjureJavaJerseyFeature.INSTANCE`?

Reply:
Changed to your suggested wording.